shared_model/bindings/CMakeLists.txt

```diff
@@ -33,7 +33,13 @@ if (SWIG_PYTHON OR SWIG_JAVA OR SWIG_CSHARP OR SWIG_NODE)
     find_package(swig REQUIRED)
     include(${SWIG_USE_FILE})
 
-    set(CMAKE_SWIG_FLAGS "")
+    if (SWIG_JAVA)
+        set(CMAKE_SWIG_FLAGS -package ${SWIG_JAVA_PKG})
+        set(CMAKE_SWIG_OUTDIR ${SWIG_JAVA_OUTDIR})     
```

Comment:
That should be set based on java package, I guess `string(REGEX REPLACE)` might do the thing